Hippo.Web/ClientApp/src/Shared/Form/FormFieldArray.tsx

```diff
@@ -0,0 +1,45 @@
+import { OrderMetadataModel, OrderModel } from "../../types";
+import FormField from "./FormField";
+import { FormFieldArrayProps } from "./formTypes";
+import { useFieldArray, useFormContext } from "react-hook-form";
+
+const FormFieldArray = <T extends Record<string, any>>({
+  control,
+  register,
+  errors,
+  arrayName = "metaData",
+}: FormFieldArrayProps<OrderModel>) => {
+  const arrayName2 = "metaData";
+  // const { control, register } = useFormContext<OrderModel>();
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: "metaData",
+  });
+
+  const keys = Object.keys(fields);
+  console.log(keys);
+
+  console.log(arrayName);
+  return (
+    <>
+      {fields.map((field, index) => {
+        console.log("field", field);
+        console.log(`[arrayName][${index}]`, [arrayName]);
+        return (
+          <>
+            <FormField
```

Comment:
You would also need to show the value field
And possibly a button to add an empty array element and Delete?